src/System.IO.UnmanagedMemoryStream/tests/UmsTests.cs

```diff
@@ -189,5 +189,67 @@ public static void CannotUseStreamAfterDispose()
                 Assert.Throws<ObjectDisposedException>(() => stream.WriteAsync(buffer, 0, buffer.Length).GetAwaiter().GetResult());
             }
         }
+
+        [Fact]
+        public static void CopyToTest()
+        {
+            byte[] testData = Enumerable.Repeat((byte)0x20, 10).ToArray();
```

Comment:
You should add `System.Linq` to project.json file and corresponding `using` directive in cs file